docs/conf.py

```diff
@@ -0,0 +1,32 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('..'))
+
+project = 'solar'
+copyright = '2024, Mark Showalter'
```

Comment:
What does this copyright mean? I suppose it doesn't affect licensing. I'm not sure I want my name in lots of software, so would it make sense to use "PDS Ring-Moon Systems Node" in its place?

Reply:
Fixed